src/main/com/example005/radek/printDiagonalsApplication.java

```diff
@@ -0,0 +1,57 @@
+package main.com.example005.radek;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+enum DiagonalType {
+    LEFT,
+    RIGHT,
+    BOTH
+}
+
+public class printDiagonalsApplication {
+
+    public static void main(String[] args) {
```

Comment:
V pripade BOTH to nevypada uplne dokonale. Zkus se na to podivat.

Reply:
Opraveno, spolu s right, počítalo to špatně